app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -344,7 +344,7 @@ private fun FavoriteParkingCard(parking: Parking, index: Int, onRemove: () -> Un
           Text(
               stringResource(
                   R.string.view_profile_screen_remove_favorite_dialog_message,
-                  parking.optName ?: ""))
+                  parking.optName ?: R.string.default_parking_name))
```

Comment:
You forget to call `stringResource` around `R.string.default_parking_name`, which is only a index (`Int`) that Koltin is probably automatically casting, which might become problematic.

Reply:
Oops, im too distracted. Glad you're here